src/components/About/Blog/BlogItem.js

```diff
@@ -142,6 +142,10 @@ const Title = styled.div`
 
 const Card = styled(_Card)`
   position: relative;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
Use @media queries to fix the height difference in mobile view as said by @elit-altum